eqt/ui/UIFormWidget.py

```diff
@@ -109,6 +109,10 @@ def addSeparator(self, name):
     def _addWidget(self, name, qwidget, qlabel=None):
         formLayout = self.uiElements['groupBoxFormLayout']
 
+        # create the default values
+        if not hasattr(self, 'widget_default'):
+            self.widget_default={}
```

Comment:
@DanicaSTFC and I discussed the benefits of separating the default state from the widget number, and store them in different places.
This would include being able to take one widget with its defaults and add to a different form easily without bringing across also widget position.

Reply:
I think the structure of how we save all of the widgets' information would be good to discuss with @paskino

Reply:
I opened a new issue to resolve this. As discussed with @lauramurgatroyd privately, this PR could be closed to resolve the save by default issue only.

Link to new issue:

https://github.com/TomographicImaging/eqt/issues/105#issue-1999238302